CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## Unreleased
+
+- Fix the behavior of the `excluded_exceptions` option: now it's used to skip capture of exceptions, not to purge the `exception` data of the event, which resulted in broken or empty chains of exceptions in reported events (#822) 
```

Comment:
When possible please wrap the lines after ~ 80 characters to keep it consistent with the rest of the changelog